sklearn/mixture/tests/test_gmm.py

```diff
@@ -97,6 +102,11 @@ def test_lmvnpdf_spherical():
     lpr = mixture.log_multivariate_normal_density(X, mu, spherecv,
```

Comment:
Travis is not happy because _naive_lmvnpdf_diag creates diagonal covariances. I think you'll need more _naive_lmvnpdf_spherical (if it exists).